lib/Config/Hooks.php

```diff
@@ -158,6 +165,35 @@ function setupWPUsers() {
     );
   }
 
+  function setupWooCommerceUsers() {
+    // WooCommerce Customers synchronization
+    add_action(
```

Comment:
We shouldn't use these WP functions directly. We should use the `\MailPoet\WP\Functions` class.

Reply:
Fixed.